### PR TITLE
fix circular dao calls, removed unused parameters, cleanup

### PR DESCRIPTION
**Pull request by Desmondmies**
added back AdminDAOTest
to get everything to work properly, `update to the latest database sql script`

### CODE REVIEW

File: edtLHD/src/main/java/fr/univtln/lhd/model/entities/users/Student.java [hunk 1]
```diff
@@ -13,14 +12,14 @@
 @Getter
 public class Student extends User {
 
-    private List<Group> studendGroup;
+    private List<Group> studentGroup;
```
Comment (Theo-Hafsaoui) [inline edtLHD/src/main/java/fr/univtln/lhd/model/entities/users/Student.java:15]: Oh yeah, I had to read that 3 time to find it

File: edtLHD/src/test/java/fr/univtln/lhd/model/entities/dao/users/StudentDAOTest.java [hunk 2]
```diff
@@ -51,7 +51,7 @@ void addNewStudent() throws SQLException {
         dao.delete(student);
     }
 
-    //@Test Need to fix circular import
+    @Test //Need to fix circular import
```
Comment (Theo-Hafsaoui) [inline edtLHD/src/test/java/fr/univtln/lhd/model/entities/dao/users/StudentDAOTest.java:54]: why keep the commentary ?

**Review by Theo-Hafsaoui — APPROVED** [review]
lgtm

File: edtLHD/src/test/java/fr/univtln/lhd/model/entities/dao/users/StudentDAOTest.java (hunk 2 above)
Comment (Desmondmies) [reply edtLHD/src/test/java/fr/univtln/lhd/model/entities/dao/users/StudentDAOTest.java:54]: forgot about it, i was just testing around and validating the test, i've forgot to remove that comment

Comment (Theo-Hafsaoui) [reply edtLHD/src/test/java/fr/univtln/lhd/model/entities/dao/users/StudentDAOTest.java]: That's remove, juste waiting for @anastygnome approval to merge

**Review by Theo-Hafsaoui — APPROVED** [review]
nice comprehensive comment

**Review by anastygnome — APPROVED** [review]
nice work

**Pull request merged**